### PR TITLE
[Serializer] Add missing XML serializer's CDATA options

### LINKED ISSUE #19851: [Serializer] Add `XmlEncoder::CDATA_WRAPPING_PATTERN` context option
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#54663
| PR author(s) | @alexpozzi
| Merged in    | 7.1

We created this issue to not forget to document this new feature. We would really appreciate if you can help us with this task. If you are not sure how to do it, please ask us and we will help you.

To fix this issue, please create a PR against the 7.1 branch in the [symfony-docs repository](https://github.com/symfony/symfony-docs).

Thank you! :smiley:

### PR DESCRIPTION
**Pull request by alexpozzi**
Fixes https://github.com/symfony/symfony-docs/issues/19851

### CODE REVIEW

**Comment by OskarStark** [comment]
https://github.com/symfony/symfony-docs/pull/18155 should be merged first

**Comment by OskarStark** [comment]
Please add a versionadded directive for your new option

**Comment by OskarStark** [comment]
> https://github.com/symfony/symfony-docs/pull/18155 should be merged first

Done, please rebase

File: components/serializer.rst [hunk 1]
```diff
@@ -1200,15 +1200,21 @@ Option                          Description
 ``remove_empty_tags``           If set to true, removes all empty tags in the      ``false``
                                 generated XML
 ``cdata_wrapping``              If set to false, will not wrap any value           ``true``
-                                containing one of the following characters (
-                                ``<``, ``>``, ``&``) in `a CDATA section`_ like
-                                following: ``<![CDATA[...]]>``
+                                matching the `cdata_wrapping_pattern`_ regex in
```
Comment (OskarStark) [inline components/serializer.rst]: where is the defined link for data_wrapping_pattern?

**Comment by OskarStark** [comment]
btw I fixed the DOCtor-RST build. I should be green after another rebase

File: components/serializer.rst (hunk 1 above)
Comment (alexpozzi) [reply components/serializer.rst]: Mistake, sorry, I didn't want to make a link

File: components/serializer.rst [hunk 2]
```diff
@@ -1200,11 +1200,21 @@ Option                          Description
...
+                                `a CDATA section`_ like following:
+                                ``<![CDATA[...]]>``
+``cdata_wrapping_pattern``      A regular expression pattern to determine if a     ``/[<>&]/``
+                                value should be wrapped in a CDATA section
 ==============================  =================================================  ==========================
 
+.. versionadded:: 6.4
+
+    The `cdata_wrapping` option was introduced in Symfony 6.4.
+
```
Comment (OskarStark) [inline components/serializer.rst:1213]:
```suggestion
```

**Comment by OskarStark** [comment]
Thank you Alessandro.

**Pull request merged**